### DISCUSSION TITLE
Integrating Swin Transformer V2 as a Backbone in MMdetection

### DISCUSSION THREAD

**Discussion by aliwaqas333** [open-mmlab/mmdetection#11600] [Ideas]
I’m currently working on enhancing object detection and instance segmentation models by incorporating the Swin Transformer V2 (Swin V2) as a backbone within the MMdetection framework. 

My question is, why hasn't Swin v2 been added yet? I can work on incorporating it, but I'm curious about the reason behind its absence.

**Comment by DIPlab56** [comment]
Salam brother, if you code for instance segmentation using mmdetection any model. Can you please share with us or me 
Thanks.....

**Comment by GeorgePearse** [comment]
It's in MMpretrain, I'm quickly checking if it's easy / simple.
